### PR TITLE
refactor(render): split up attach + render concerns

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Related to https://github.com/camunda/camunda-modeler/issues/2374

Rationale: https://github.com/camunda/camunda-modeler/issues/2374#issuecomment-883171791

### CODE REVIEW

File: src/render/BpmnPropertiesPanelRenderer.js [hunk 1]
```diff
@@ -20,32 +21,31 @@ export default class BpmnPropertiesPanelRenderer {
 
   constructor(config, injector, eventBus) {
     let {
```
Comment (MaxTru) [inline src/render/BpmnPropertiesPanelRenderer.js:23]: (minor) Can be `const`

**Review by MaxTru — COMMENTED** [review]
What do you think about adding an integration testCase:
`should ensure creating  + attaching -> import -> detach -> attach works keeping state`

**Comment by pinussilvestrus** [comment]
An integration test definitely makes sense 👍

File: src/render/BpmnPropertiesPanelRenderer.js [hunk 2]
```diff
@@ -123,7 +119,13 @@ export default class BpmnPropertiesPanelRenderer {
   }
 
   _render(element) {
-    if (!element || isImplicitRoot(element)) {
+    const canvas = this._injector.get('canvas');
```
[the remark this answers is not in the document]
Comment (pinussilvestrus) [reply src/render/BpmnPropertiesPanelRenderer.js:122]: As a personal taste, I like to have variable declaration before any `if` clause when possible. 🙂

**Comment by pinussilvestrus** [comment]
I added a simple test case to verify the state keeps during detach & attach (cf. https://github.com/bpmn-io/bpmn-properties-panel/pull/40/commits/8113827f7a93cf821434d94944054850e167674e)

**Review by MaxTru — APPROVED** [review]
cool stuff!

**Pull request merged**